src/main/java/com/zs/exc5/Orders.java

```diff
@@ -0,0 +1,237 @@
+     * @throws SQLException Throwing SQLException.
+     */
+    public void lastOrder() throws SQLException {
+        logr.info("Enter userId");
+        String uid = scan.next();
+        logr.info("Enter value of x");
+        int x = scan.nextInt();
+        String str = "select * from orders_detail where userid = ? order by orderdate desc limit  ? ;";
+        stmt = c.prepareStatement(str);
+        stmt.setString(1, uid);
+        stmt.setInt(2, x);
+        rs = stmt.executeQuery();
+        while (rs.next()) {
+
+            logr.info(rs.getString(1) + "\t" + rs.getDate(2) + "\t" + rs.getString(3) + "\t" + rs.getString(6));
+        }
+
+
+    }
+
+    /**
+     * This function is updating data in order details.
+     * @throws SQLException
+     */
+    public void updateOrder() throws SQLException {
+        String feild1, feild2;
+        String val1, val2;
+        logr.info("enter feild to be updated and field have condtion");
+        feild1 = scan.next();
+        feild2 = scan.next();
+        logr.info("Enter new value and value have condition ");
+        val1 = scan.next();
+        val2 = scan.next();
+        String str = "UPDATE orders_detail SET " + feild1 + "= ?  FROM place_orders WHERE orders_detail.orderid=place_orders.orderid AND orders_detail." + feild2 + " = ? ;";
+        stmt = c.prepareStatement(str);
+        stmt.setString(1, val1);
+        stmt.setString(2, val2);
+        int result = stmt.executeUpdate();
+        if (result == 1)
+            logr.info("Successfully Updated");
+        else
+            logr.info("Not Successfully Updated");
+
+    }
+    /**
+     *   This function is deleting data from order table.
+     * @throws SQLException Throwing SQLException.
+     */
+    public void deleteOrder() throws SQLException {
+        String feild1, val;
+        logr.info("Enter feild and value making condition to delete");
+        feild1 = scan.next();
+        val = scan.next();
+        String str = "delete from orders_detail using  place_orders where orders_detail.orderid=place_orders.orderid AND orders_detail." + feild1 + " = ? ;";
+        stmt = c.prepareStatement(str);
+        stmt.setString(1, val);
+        int result = stmt.executeUpdate();
+        if (result == 1)
+            logr.info("Successfully Deleted");
+        else
+            logr.info("Not Successfully Deleted");
+
+    }
+    /**
+     *   This function is displaying data of Order table.
+     */
+    public void selectOrder() throws SQLException {
+        String str = "select od.orderid ,userid,productid,source_address,destination_address from orders_detail od INNER JOIN place_orders po ON od.orderid=po.orderid ;";
+        stmt = c.prepareStatement(str);
+        rs = stmt.executeQuery();
+        while (rs.next()) {
+
+            logr.info(rs.getString(1) + "\t" + rs.getString(2) + "\t" + rs.getString(3) + "\t" + rs.getString(4) + "\t" + rs.getString(5));
```

Comment:
Instead of using rs.getString you can also use a proper mapper class to map these data. That would be more meaningful while reading the code.